templates/ansible/resource.erb

```diff
@@ -272,11 +269,11 @@ def resource_to_request(module):
 <%
   encoder_name = if object.encoder?
                    object.transport.encoder
-                 else config['encoder']
-                   config['encoder']
+                 else object.encoder
```

Comment:
Why is there a condition for a else statement here?

Reply:
Ping

Reply:
(But actually fixed now)